loom/script/native/lsNativeInterface.cpp

```diff
@@ -542,6 +542,8 @@ void lualoom_newnativeuserdata(lua_State *L, NativeTypeBase *nativeType, void *p
     // set the user data's native type
     ud->m_nativeType = nativeType;
 
+    ud->m_owner = owner;
```

Comment:
I think I did? At the definition at least. Should I copy it to all the functions that accept it as well?